doc/wiki/Contribute-related/Contribute:-Update-Wiki.md

```diff
@@ -0,0 +1,160 @@
+   - Clone your forked Wiki git. You may find the Wiki URL at the bottom right of the Wiki page.
+
+     [[images/WikiGitLocation.png]]
+
+   - Edit by `gollum`.
+     * NOTE: To use `gollum` for a branch different from `master` (say `new_branch`), one needs to do `gollum --ref new_branch`.
+     * `gollum --lenient-tag-lookup /path/to/wiki`
+     * Open `http://localhost:4567` in your browser.
+   - Push your changes to the forked Wiki git
+   - **Copy wiki to doc**
+
+     This step is like `git push` to your branch.
+     * Click `Action` > `Workflows` > `Copy wiki to doc` > `Run workflow` > choose `new_contribution_branch` branch > Click green `Run workflow`. Once the workflow is done, the docs in the specific branch are updated to the latest Wiki.
+
+     [[images/CopyWikiToNewBranch.png]]
+
+1. **From GitHub website**
+
+   We treat the Wiki page on GitHub as a local in this method, and we use the workflow to approach the `pull` and `push` behavior of the Wiki.
+   - **Copy wiki doc to wiki page**
+
+     This step is like `git checkout new_contribution_branch` but the GitHub Wiki website version.
+     * Click `Action` > `Workflows` > `Copy doc to wiki` > `Run workflow` > choose `new_contribution_branch` branch > Click green `Run workflow`. Once the workflow is done, the Wiki is also updated to the `new_contribution_branch` branch version.
+
+      [[images/CopyDocFromNewBranch.png]]
+
+   - **Edit on GitHub Wiki website**
+
+      See [About wikis - GitHub Docs](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis) for more details.
+   - **Copy wiki to doc**
+
+     This step is like `git push` to your branch.
+     * Click `Action` > `Workflows` > `Sync wiki to doc` > `Run workflow` > choose `new_contribution_branch` branch > Click green `Run workflow`. Once the workflow is done, the docs in the `new_contribution_branch` branch are updated to the latest wiki.
+
+      [[images/CopyWikiToNewBranch.png]]
+
+1. **On the local terminal**
+
+   Just like how you edit the code on the local terminal. The files of Wiki can be found at `docs/wiki`. Once you push your changes to `origin`, you may preview your changes on the Wiki page by doing:
+   * Click `Action` > `Workflows` > `Copy doc to wiki` > `Run workflow` > choose `new_contribution_branch` branch > Click green `Run workflow`. Once the workflow is done, the Wiki is also updated to the `new_contribution_branch` branch version.
+
+## Contribute
+   Before you file a new PR, please check that all the hyperlinks work, the images are clear enough, and all the related pages about the PR have been updated.
+   1. File a PR to the GAMER public version.
+      * NOTE: The contribution is made through the files in the doc (e.g. `doc/wiki/`), not through the Wiki page of the forked repository.
```

Comment:
```suggestion
   1. Before filing a new PR, please ensure the following:
     * All hyperlinks are functional.
     * Images are clear.
     * All related pages affected by this PR have been updated.
   2. File a PR to the public GAMER repository.
     * NOTE: Contributions should be made through the files in `gamer/doc/wiki/` rather than the Wiki pages.
```